### ISSUE TITLE
[Replay UI] German Text Incorrect and Unable to fit into Button

### ISSUE THREAD

**Issue by knyghtmare** [wesnoth/wesnoth#7524]
![image](https://user-images.githubusercontent.com/5283677/231146069-d130d6e6-92e0-4205-8162-b950144f805e.png)

Observed on 1.17.14+dev (latest commit)

**Comment by Wedge009** [comment]
Isn't this just yet another duplicate of #2575 and related issues?

**Comment by gfgtdf** [comment]
> Isn't this just yet another duplicate of https://github.com/wesnoth/wesnoth/issues/2575 and related issues?

From looking at the screenshots, the situation got worse though, There the buttons just overlap, here the buttontext also i not longer on the button.

**Comment by knyghtmare** [comment]
> In this case it is additionally a translation error though.

I guess it's not a duplicate then.

**Comment by CelticMinstrel** [comment]
> Seems so. Add German to the list of affected languages then. Might be more though..maybe Nihongo and Tagalog?

I wouldn't expect this to affect Japanese since it commonly has _shorter_ translations due to the use of kanji… that said, if there's no kanji for a term, perhaps it could be affected.

Ultimately the solution is to use the GUI2 layout system, but it might be awhile before we can get there. An icon would also be a good solution. Then any text goes into a tooltip which should hopefully automatically size itself appropriately.

**Comment by gfgtdf** [comment]
> I guess it's not a duplicate then.

Agreed at least for, this part

**Comment by Wedge009** [comment]
I didn't think we catalogued translation errors here - shouldn't that go to the translation maintainers?

**Comment by Wedge009** [comment]
Renamed to 'distinguish' from #2575 (which already mentions German).

**Comment by gfgtdf** [comment]
> I didn't think we catalogued translation errors here - shouldn't that go to the translation maintainers?

I don't know what the procedure for translations is tbh :(, I could edit the .po file myself and commit but i'm unsure whether that's the correct way to do it.

**Comment by Wedge009** [comment]
I submit everything through @ivanovic as per https://wiki.wesnoth.org/WesnothTranslationsHowTo.

For Deutsch, you'd probably need to submit to @Byteron as per https://wiki.wesnoth.org/GermanTranslation, although it seems to list @stevecotton as a translator as well.

**Comment by stevecotton** [comment]
The current way to submit translations is to discuss in Discord / IRC. German is a bit odd in that it's been working well enough that no-one's properly agreed a method for it; I guess @gfgtdf would push them directly to Git after chatting to @Byteron and @sevu.

While I'm glad that someone added me to the list of translators, I'm just doing basic cleanups, mainly related to things that I've changed the English source for; my knowledge of German is no where near the level to be the maintainer.

**Comment by sevu** [comment]
Translation has been updated
